Cargo.toml

```diff
@@ -9,6 +9,7 @@ documentation = "https://docs.rs/hashlink"
 readme = "README.md"
 keywords = ["data-structures", "no_std"]
 license = "MIT OR Apache-2.0"
+rust-version = "1.65"
```

Comment:
`cargo clippy --fix` had started using `hash_one()` which stabilized in 1.71 -- I think setting this will help prevent that sort of thing.

Reply:
That's just good to have anyway, thank you!